Gems/Atom/RHI/Vulkan/Code/Source/RHI/ImagePool.cpp

```diff
@@ -49,7 +49,8 @@ namespace AZ
             uint32_t memoryTypeBits = 0;
             {
                 // Use an image descriptor of size 1x1 to get the memory requirements.
-                RHI::ImageDescriptor imageDescriptor = RHI::ImageDescriptor::Create2D(descriptorBase.m_bindFlags, 1, 1, RHI::Format::R8G8B8A8_UNORM);
+                RHI::ImageBindFlags bindFlags = RHI::ImageBindFlags::ShaderReadWrite | RHI::ImageBindFlags::CopyWrite;
```

Comment:
Why don't we use the m_bindFlags (or ORring with them)?

Reply:
So the m_bindFlags from the pool may contain both RHI::ImageBindFlags::Color and RHI::ImageBindFlags::DepthStencil.
The vk GetImageMemoryRequirements function won't work with both flags active. 

Basically, no image can be created with both flags.